internal/engine/live_update_build_and_deployer.go

```diff
@@ -159,20 +160,40 @@ func (lubad *LiveUpdateBuildAndDeployer) buildAndDeploy(ctx context.Context, cu
 	}()
 
 	if len(toArchive) > 0 {
-		l.Infof("Will copy %d file(s) to container: %s", len(toArchive), cInfo.ContainerID.ShortStr())
+		l.Infof("Will copy %d file(s) to container(s): %s", len(toArchive), cIDStr)
 		for _, pm := range toArchive {
 			l.Infof("- %s", pm.PrettyStr())
 		}
 	}
 
-	err = cu.UpdateContainer(ctx, cInfo, pr, build.PathMappingsToContainerPaths(toRemove), boiledSteps, hotReload)
-	if err != nil {
-		if build.IsUserBuildFailure(err) {
-			return WrapDontFallBackError(err)
+	var lastUserBuildFailure error
+	for _, cInfo := range state.RunningContainers {
+		err = cu.UpdateContainer(ctx, cInfo, pr, build.PathMappingsToContainerPaths(toRemove), boiledSteps, hotReload)
+		if err != nil {
+			if build.IsUserRunFailure(err) {
+				// Keep running updates -- we want all containers to have the same files on them
+				// even if the Runs don't succeed
+				lastUserBuildFailure = err
+				logger.Get(ctx).Infof("  → FAILED TO UPDATE CONTAINER %s with user run error: %v", cInfo.ContainerID, err)
```

Comment:
"user run error" seems like a weird thing to say to the user. This is something they expect to happen frequently in normal operation, right?